profile/README.md

```diff
@@ -1,13 +1,20 @@
-# Single sign-on is coming to the B.C. government’s GitHub organizations 
+### Welcome to the B.C. Government’s official GitHub Organization!
 
-:sparkles: The Developer Experience team has enabled Single Sign-On (SSO) for GitHub. We’re starting with the bcgov GitHub organization. After we finish transitioning bcgov, we’ll enable other OCIO-managed GitHub organizations.
+<img align="right" width="250" height="250" src="images/Developer.png">
 
-By **July 30, 2024**, all bcgov GitHub organization users must link their IDIRs with their GitHub IDs. If this isn’t done by the deadline, access to the GitHub organization will be automatically removed. 
+The B.C. Government commits to [open ways of working](https://digital.gov.bc.ca/policies-standards/dcop/open/) to transform into a Digital Government. We leverage GitHub to achieve our objective. Our open practices let us to share source code and non-sensitive data with our government developer community and partners.
 
-Contractors without IDIRs must order one before SSO can be enabled.
+Within the Ministry of Citizens’ Services, the Developer Experience team’s purpose is offering the best support possible to developer teams. We centralize, streamline and innovate developer resources so we can contribute towards quick project starts, and facilitate continuous improvement. 
 
-[Learn more about IDIRs and technical information.](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide?utm_source=guide-idir&utm_medium=github&utm_campaign=bcgov-readme#idirs)
+Although developers work deeper in the government ecosystems, their deliverables are crucial to building public trust. Whether top-down or collaborative, developers finalize a vision by building products that both government workers and public can interact with. The public can be impacted by products and services immediately, or through a few degrees of separation.
+
+### Joining our community 
+<img align="right" width="250" height="250" src="images/Programmer.pngg">
+
+1. [Authentiacate your IDIR](https://github.com/orgs/bcgov/sso)
+2. You'll get an email with an invitation to join the **bcgov** GitHub organization.
```

Comment:
```suggestion
2. You'll get an email with an invitation to join the **bcgov** GitHub organization. The invitation will be sent to the email address associated with your GitHub account.
```

May want to put a blurb in where to expect the invitation. We've had people in the past confused about which email address the invitation would be sent too.